project.go

```diff
@@ -27,7 +27,7 @@ func (s *ProjectClient) ListProjects(opt ListProjectsOptions) ([]Project, error)
 // Check if the project name provided already exist
 func (s *ProjectClient) CheckProject(projectName string) error {
 	resp, _, errs := s.NewRequest(gorequest.HEAD, "").
-		Query(fmt.Sprintf("project_name=%s", projectName)).
+		Query(map[string]string{"project_name": url.PathEscape(projectName)}).
```

Comment:
I would actually expect `gorequest` to escape query parameters on its own in this case; maybe check, just to make sure that there is no double-escaping.

In any way, the escape function to use in this case would be `url.QueryEscape`, not `url.PathEscape` ;)

Reply:
Doesn't seem like gorequest does it, although i might be wrong. As we're querying a map `gorequest` passes it's content down to the following:
https://github.com/parnurzeal/gorequest/blob/a578a48e8d6ca8b01a3b18314c43c6716bb5f5a3/gorequest.go#L386

Reply:
I think it's alright:

https://github.com/parnurzeal/gorequest/blob/a578a48e8d6ca8b01a3b18314c43c6716bb5f5a3/gorequest.go#L1205-L1211

Basically, it passes all values into a `url.Values`, which does URL-encoding on its own. So no need to double-escape.